roles/change-paths/handlers/main.yml

```diff
@@ -7,3 +7,4 @@
     password: "{{ acquia_api_key }}"
     force_basic_auth: yes
   with_items: "{{ site_id }}"
+  ignore_errors: yes
```

Comment:
Why ignore errors here?

Reply:
I was getting errors on cache clear and `no1curr`. It's probably resolved by the cache-per-host setting on the servers now but still, `no1curr` if there's an error on cache clear. I don't want to stop the migration for that.